packages/block-library/src/media-text/editor.scss

```diff
@@ -26,11 +26,16 @@
 	width: 100% !important;
 }
 
-.wp-block-media-text.is-image-fill .editor-media-container__resizer {
+.wp-block-media-text.is-image-fill .editor-media-container__resizer,
+.wp-block-media-text.is-image-fill .components-placeholder.has-illustration {
 	// The resizer sets an inline height but for the image fill we set it to full height.
 	height: 100% !important;
 }
 
 .wp-block-media-text > .block-editor-block-list__layout > .block-editor-block-list__block {
 	max-width: unset;
 }
+
+.wp-block-media-text--placeholder-image {
+	min-height: 205px;
```

Comment:
What is 205?

Reply:
I am trying to match the height of the media area placeholder that shows the three options: upload, media library, featured image.
That way, if no featured image is set, and the featured image option is selected, there will not be a shift in height.
When all three options are available, the height of it is "205.59"

Reply:
let's add a comment explaining that, for future reference